lib/tmx-parser/listener.rb

```diff
@@ -0,0 +1,80 @@
+# encoding: UTF-8
+
+module TmxParser
+
+  class Listener
+    include TagNames
+
+    attr_reader :units, :proc
```

Comment:
I don't think proc is a keyword? The `Proc` constant is how you create new procs, eg `Proc.new`.